app/schemas.py

```diff
@@ -418,7 +461,8 @@ def remove_content_for_non_broadcast_templates(self, template, **kwargs):
 class TemplateHistorySchema(BaseSchema):
     reply_to = fields.Method("get_reply_to", allow_none=True)
     reply_to_text = fields.Method("get_reply_to_text", allow_none=True)
-    process_type = field_for(models.Template, "process_type")
+    process_type = SafeEnumField(TemplateProcessType, by_value=True)
```

Comment:
Nice!  What does `SafeEnumField` do for us here?  I hadn't come across that, let alone `marshmallow_enum`!